src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -96,22 +96,22 @@ public static Email parseEmail(String email) throws ParseException {
     }
 
     /**
-     * Parses a {@code String studyLevel} into a {@code studyLevel}.
+     * Parses a {@code String studyLevel} into a {@code String studyLevel}.
```

Comment:
I'm assuming the method is named `parseStudyLevel` to be uniform with the rest. However, for study level, it isn't exactly '_parsing_', since it's `String` -> `String`. As compared to say email, where it's `String` -> `Email`.

Perhaps without changing method name, we could change the description to "Sanitize/Clean studyLevel input"? or something along that line. Don't think it's a big issue tbh, but what do y'all think?

Same goes for relationship as well!

Reply:
Yup! I agree with what you said, idm changing to what you suggested instead. Personally, I think code clarity is more important than uniformity in this case.